plugins/src/plugins/gripper/gripper.cpp

```diff
@@ -92,12 +102,14 @@ void Gripper::OnUpdate(const common::UpdateInfo & /*_info*/)
     this->open();
   }
   last_action_rcvd_ = NOTHING;
+  if(grippedPuck && test_probability(PROB_PUCK_FALLS, delta_t))this->open(); //sometimes let the puck fall down while walking around with it
```

Comment:
Please fix the formatting, this should be something like
```
if(grippedPuck && test_probability(PROB_PUCK_FALLS, delta_t)) {
  // sometimes let the puck fall down while walking around with it
  this->open();
}
```

Reply:
Here, more output would be helpful. A short info message (e.g., "Random failure: dropping puck") will be sufficient